portal/settings.py

```diff
@@ -1,39 +1,138 @@
+
+# # PASSWORD VALIDATORS
+
+# AUTH_PASSWORD_VALIDATORS = [ {"NAME": "django.contrib.auth.password_validation.UserAttributeSimilarityValidator"}, {"NAME": "django.contrib.auth.password_validation.MinimumLengthValidator"}, {"NAME": "django.contrib.auth.password_validation.CommonPasswordValidator"}, {"NAME": "django.contrib.auth.password_validation.NumericPasswordValidator"}, ]
+
+# # INTERNATIONALIZATION
+
+# LANGUAGE_CODE = "en-us" 
+# TIME_ZONE = "UTC" 
+# USE_I18N = True 
+# USE_TZ = True
+
+# # STATIC FILES
+
+# STATIC_URL = "static/" 
+# STATIC_ROOT = BASE_DIR / "staticroot" 
+# STATICFILES_STORAGE = "whitenoise.storage.CompressedManifestStaticFilesStorage"
+
+# # DEFAULT AUTO FIELD
+
+# DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
+
+# # AUTHENTICATION BACKENDS
+
+# AUTHENTICATION_BACKENDS = [ "django.contrib.auth.backends.ModelBackend", "allauth.account.auth_backends.AuthenticationBackend", ]
+
+# # SITE ID
+
+# SITE_ID = 1
+
+# # DJANGO-ALLAUTH SETTINGS
+
+# ACCOUNT_EMAIL_VERIFICATION = "mandatory" 
+# ACCOUNT_SIGNUP_FIELDS = ["email", "username", "password1", "password2"] 
+# ACCOUNT_LOGIN_METHODS = "email" 
+# ACCOUNT_MAX_EMAIL_ADDRESSES = 3 
+# ACCOUNT_FORMS = {"signup": "portal.forms.CustomSignupForm"}
+
+# # EMAIL CONFIGURATION
+
+# if os.getenv("DJANGO_EMAIL_HOST"): 
+#     EMAIL_BACKEND = "django.core.mail.backends.smtp.EmailBackend" 
+#     EMAIL_HOST = os.getenv("DJANGO_EMAIL_HOST") 
+#     EMAIL_PORT = int(os.getenv("DJANGO_EMAIL_PORT", 587)) 
+#     EMAIL_HOST_USER = os.getenv("DJANGO_EMAIL_HOST_USER") 
+#     EMAIL_HOST_PASSWORD = os.getenv("DJANGO_EMAIL_HOST_PASSWORD") 
+#     EMAIL_USE_TLS = os.getenv("DJANGO_EMAIL_USE_TLS", "True").lower() in ("true", "1") 
+#     DEFAULT_FROM_EMAIL = os.getenv("DJANGO_DEFAULT_FROM_EMAIL") 
+# else: EMAIL_BACKEND = "django.core.mail.backends.console.EmailBackend"
+
+
+
+
+
+
+
+
+
+
+
 
-Generated by 'django-admin startproject' using Django 5.1.7.
 
-For more information on this file, see
-https://docs.djangoproject.com/en/5.1/topics/settings/
 
-For the full list of settings and their values, see
-https://docs.djangoproject.com/en/5.1/ref/settings/
-"""
 
 import os
 from pathlib import Path
-
+from dotenv import load_dotenv
 import dj_database_url
 
+load_dotenv()
+
+
 # Build paths inside the project like this: BASE_DIR / 'subdir'.
 BASE_DIR = Path(__file__).resolve().parent.parent
 
-
 # Quick-start development settings - unsuitable for production
 # See https://docs.djangoproject.com/en/5.1/howto/deployment/checklist/
-
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = os.environ.get("SECRET_KEY")
+# SECRET_KEY = os.environ.get("SECRET_KEY")
+SECRET_KEY = os.environ.get("SECRET_KEY","h7Qap2XP9-cNEsRpXv7GN1KWIMYHnK4xcEdhRk3_2MXBfyV3b_yK53brLhs7TwFsvpo")
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = bool(os.environ.get("DEBUG", default=0))
+# DEBUG = bool(os.environ.get("DEBUG", default=0))
+DEBUG =True
```

Comment:
why we need to set `DEBUG` to true while in the previous logic the user can it however it can